plugins/module_utils/nae.py

```diff
@@ -392,13 +439,19 @@ def show_pre_change_analyses(self):
         self.result['Analyses'] = result
         return result
 
-    def is_json(self, myjson):
+    def is_json(self, filename):
```

Comment:
should we keep the is_json function to take text input and create a is_json_file to do the file opening and test with is_json function?

Same for XML

Reply:
I actually already changed it and was gonna be part of the next commit :)

Reply:
I do not see the next commit you are talking about so should we still make that change?